docs/basics/integrations/deque.md

```diff
@@ -30,23 +31,23 @@ Check out the [Deque Accessibility Guide](https://www.deque.com/web-accessibilit
 
 ## Sauce Labs UI
 
-Sauce Labs will display the accessibility results in our UI as a tab in the Job itself:
+Sauce Labs will display the accessibility results in our UI as a tab in the job itself:
 
 <img src={useBaseUrl('img/accessibility/example.png')} alt="Accessibility Example in Sauce UI"/>
 
 
 ## Language Specific Examples
 
 <Tabs
-defaultValue="java"
-values={[
-{ label: 'Java', value: 'java', },
-{ label: 'WebdriverIO', value: 'wdio', },
-{ label: 'Python', value: 'python', },
-{ label: 'Ruby', value: 'ruby', },
-{ label: 'C#', value: 'dotnet', }
-]
-}>
+  groupId="axe-lang-examples"
+  defaultValue="java"
+  values={[
+    {label: 'Java', value: 'java'},
+    {label: 'WebdriverIO', value: 'webdriver'},
+    {label: 'Python', value: 'python'},
+    {label: 'Ruby', value: 'ruby'},
+    {label: 'C#', value: 'c#'},
```

Comment:
agreed we shouldn't be using dotnet (which is how Selenium refers to its bindings for some reason)
We should use either `cs` or `csharp` to keep it inline with the highlight tags https://rdmd.readme.io/docs/code-blocks#language-support

Reply:
that is good to know!

Reply:
Changed to `csharp`